indexing-service/src/main/java/org/apache/druid/indexing/common/tasklogs/ConsoleLoggingEnforcementConfigurationFactory.java

```diff
@@ -112,36 +115,38 @@ protected void doConfigure()
       // If not, replace it's appender to ConsoleAppender.
       //
       for (LoggerConfig logger : loggerConfigList) {
-        applyConsoleAppender(logger, consoleAppender);
+        applyConsoleAppender(logger, consoleAppenders);
       }
     }
 
-    @Nullable
-    private Appender findConsoleAppender()
+    @NotNull
```

Comment:
I think this should be `@Nonnull` rather than `@NotNull` as there's no validation

Reply:
Good catch. Thanks